src/client/components/hiluk-menu/hiluk-menu.tsx

```diff
@@ -41,6 +47,7 @@ export interface HilukMenuProps {
 export interface HilukMenuState {
   url?: string;
   fixedTimeUrl?: string;
+  openExternalSystemView?: boolean;
```

Comment:
This boolean value has really three states - true, false and null - why? What's the difference between false and empty value?